### PR TITLE
Bugfix testing feedback anonymous

### PR DESCRIPTION
**Pull request by koepferd**
## Proposed Changes

  - authToken refreshing for `<WaitingRoom />`
  - set enquiries empty with new tab functionality
  - session read fix for ended chats

### CODE REVIEW

File: src/components/sessionsList/SessionsList.tsx [hunk 1]
```diff
@@ -400,10 +400,12 @@ export const SessionsList: React.FC = () => {
 					) {
 						setIsLoading(false);
 						setIsReloadButtonVisible(true);
+						console.log('emtäää 1');
```
Comment (iam-robin) [inline src/components/sessionsList/SessionsList.tsx]: 😁 please remove log

File: src/components/sessionsList/SessionsList.tsx [hunk 2]
```diff
@@ -400,10 +400,12 @@ export const SessionsList: React.FC = () => {
...
 						reject(FETCH_ERRORS.EMPTY);
 					} else if (error.message === FETCH_ERRORS.EMPTY) {
 						setIsLoading(false);
 						setHasNoSessions(true);
+						console.log('emtäää 2');
```
Comment (iam-robin) [inline src/components/sessionsList/SessionsList.tsx]: same

Comment (koepferd) [reply src/components/sessionsList/SessionsList.tsx]: ...

File: src/components/sessionsList/SessionsList.tsx (hunk 1 above)
Comment (koepferd) [reply src/components/sessionsList/SessionsList.tsx]: ...

**Review by iam-robin — APPROVED** [review]

**Pull request merged**